src/sentinel.c

```diff
@@ -4317,15 +4319,25 @@ void sentinelSetCommand(client *c) {
         }
     }
 
-    if (changes) sentinelFlushConfig();
+    if (changes) {
+        if (sentinelFlushConfig() == C_ERR){
+            addReplyErrorFormat(c,"WARNING: new configurations save on disk failed due to some reconfiguration file reasons.!!!");
```

Comment:
My comment regarding serverLog(LL_WARNING,..) above is also relevant to addReplyErrorFormat()